### DISCUSSION TITLE
k-mer coverage computation

### DISCUSSION THREAD

**Discussion by apcamargo** [ablab/spades#1352] [Q&A] [answered]
The reported average k-mer coverage depths that are reported in the assembly graphs seem to be computed as: $\text{No. k-mers} \div (\text{Segment length} - \text{k-mer length})$. I'm trying to understand why that is the case.

If the number of k-mers in a sequence is $(\text{Segment length} - \text{k-mer length} + 1)$, why is $(\text{Segment length} - \text{k-mer length})$ the denominator?

**Comment by asl** [comment, marked as answer]
Because k-mers are overlaps. The sequences are broken into (k+1)-mers
